robyn/__init__.py

```diff
@@ -191,12 +191,13 @@ def startup_handler(self, handler: Callable) -> None:
     def shutdown_handler(self, handler: Callable) -> None:
         self._add_event_handler(Events.SHUTDOWN, handler)
 
-    def is_port_in_use(self, port: int) -> bool:
+    def get_socket(self, url: str, port: int) -> (bool, SocketHeld):
```

Comment:
Also could you verify how the test performs with multiple --workers and --processes?

Reply:
how exactly? I tried running `robyn app.py --processes 2 --workers 4` and that seemed to work fine.

Reply:
Alright, that sounds good to me.